### PR TITLE
Madeline Rule edits 05/13/2024

### PR DESCRIPTION
**Pull request by mrule**
## Description:
* Updated front matter on index.md
* Moved page description on index.md
* Added numbers to integration steps on integrationsteps.md and getuserprofile.md
* Added indent after numbered steps on integrationsteps.md and getuserprofile.md
* Changed wording of integrations steps on integrationsteps.md and getuserprofile.md

### Resolves:

N/A

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md](https://github.com/okta/okta-developer-docs/blob/7f2b3e590bdf8b3addae6b7271d1e044cd6cc084/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Findex.md) | [91](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fafba5f9-5a85-49ea-bc32-5f88ae9d08ee) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/getuserprofile.md](https://github.com/okta/okta-developer-docs/blob/7f2b3e590bdf8b3addae6b7271d1e044cd6cc084/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fgetuserprofile.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5a571661-e6b0-410a-a682-0ebaee5d7e21) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md](https://github.com/okta/okta-developer-docs/blob/7f2b3e590bdf8b3addae6b7271d1e044cd6cc084/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fintegrationsteps.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f8b11ed0-dabb-45dd-a7d4-1f8b5366cddf) | 61  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.53491443-7589-4586-ad60-020d5baeb0ca?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/getuserprofile.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-## Get the user profile information
+### 4: Get the user profile information
 
-After the user signs in successfully, request basic user information from the authorization server using the tokens that were returned in the previous step.
+   Upon successful user sign-in, fetch basic user information from the authorization server using the tokens obtained in the previous step.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/getuserprofile.md:3]: our style is to not use sign-in as a noun. That doesn't show up in the Git wiki page. So, no way of knowing that.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/getuserprofile.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
-```javascript
+   ```javascript
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/getuserprofile.md:5]: why indent?

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
----
-title: Basic sign-in flow with the password factor
+--- 
+title: Basic sign-in flow with password-only
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md:2]: Factor is used to call out that it's a "factor" - which we have many at Okta. If this were changed at all, we'd just remove "factor".

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
 ---
 
 <ApiLifecycle access="ie" />
 
-Enable a password-only sign-in flow in your web app using the embedded SDK.
-
 > **Note**: Passwords are a security vulnerability because they can be easily stolen and are prone to phishing attacks. Give your users the ability to use other authenticators by replacing password-only sign-in experiences with either a [password-optional](https://developer.okta.com/docs/guides/pwd-optional-overview) or a multifactor experience.
 <StackSnippet snippet="pwdoptionalusecase" />
 
+Enable a password-only sign-in flow in your web app using the embedded SDK.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md:12]: why move this?

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md [hunk 5]
```diff
@@ -1,36 +1,36 @@
-### Your app displays the sign-in page
+### 1: Your app displays the sign-in page
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md:1]: Our style is to not number headings. Also something that is in the more in-depth style guide and not on the Git wiki

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md [hunk 6]
```diff
@@ -1,36 +1,36 @@
...
 
-Build a sign-in page that captures both the user's name and their password.
+   Build a sign-in page that captures both the user's username and password.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md:3]: Their username may be something completely different than their name. So, we specify here in a conversational tone: user's name and their password.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md [hunk 7]
```diff
@@ -1,36 +1,36 @@
...
+   ```javascript
+   const authClient = getAuthClient(req);
+   const transaction = await authClient.idx.authenticate({ username, password });
+   ```
 
-### Your app handles an authentication success response
+### 3: Your app handles an authentication response
 
-`authenticate()` returns a `transaction` object with a `status` property indicating the current state of the sign-in flow. Handle the returned `IdxStatus` value accordingly:
+   Use `authenticate()` to get a `transaction` object containing the current sign-in flow status. Handle the `IdxStatus` value returned accordingly:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md:27]: The dev isn't "using" `authenticate()` here - this section is talking about how to handle items returned in the response that comes back after `authenticate()` is used in the previous section.


We need to call out the specific `status` property here. What we don't want happening is the dev searching the response for what that "current sign-in flow status" might be called in the response. The original text calls that out for them.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md [hunk 8]
```diff
@@ -1,36 +1,36 @@
...
 
-#### Success status
+   #### Success status
 
-When the user correctly supplies their password, `IdxStatus` equals `IdxStatus.SUCCESS`. Call `tokenManager.setTokens()` to save the tokens retrieved from the response for future requests, and then redirect the user back to the home page. The user is now signed in.
+   When the user correctly inputs their password, `IdxStatus` equals `IdxStatus.SUCCESS`. Call `tokenManager.setTokens()` to save the tokens retrieved from the response for future requests, then redirect the user back to the homepage. The user is now signed in.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md:31]: the user may "enter" their password or supply their password....... I don't think "inputs" is an appropriate edit here. Not something that is said often in conversation. The other wording is more conversational.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md [hunk 9]
```diff
@@ -45,13 +45,13 @@ When the user correctly supplies their password, `IdxStatus` equals `IdxStatus.S
...
    // Handle other statuses
 }
-```
+   ```
 
-#### Other authentication statuses
+   #### Other authentication statuses
 
-Handle other returned `IdxStatus` cases if the user didn't sign in successfully or there are other factors to verify. For example:
+   Handle other `IdxStatus` cases as follows:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md:52]: calling out that this is "returned" is helpful. Altho not worded that well in the original - they are providing examples of other "returned `IdxStatus` cases"

**Review by susanharper-okta — COMMENTED** [review]
comments

**Pull request closed without merging**